packages/dd-trace/src/plugins/util/web.js

```diff
@@ -432,6 +433,15 @@ function addRequestTags (context) {
     [HTTP_USERAGENT]: req.headers['user-agent']
   })
 
+  // if client ip has already been set by appsec, no need to run it again
+  if (config.clientIpEnabled && !span.context()._tags.hasOwnProperty(tags.HTTP_CLIENT_IP)) {
+    const clientIp = extractIp(config, req)
+
+    if (clientIp) {
+      span.setTag(tags.HTTP_CLIENT_IP, clientIp)
```

Comment:
Just for the sake of consistency, you could have the tags constant be "extracted" from the `tags` variable, just like you did in the test file. Look at line 16-28 of web.js and you'll see what I mean.
```suggestion
      span.setTag(HTTP_CLIENT_IP, clientIp)
```